tutorials/terminology.md

```diff
@@ -10,19 +10,19 @@ to developers touching the source code.
 
 * **Entity**: Every "object" in the world, such as models, links,
     collisions, visuals, lights, joints, etc.
-    An entity [is just a numeric ID](namespace gz_1_1gazebo.html#ad83694d867b0e3a9446b535b5dfd208d),
+    An entity [is just a numeric ID](namespacegz_1_1sim.html#ad83694d867b0e3a9446b535b5dfd208d),
```

Comment:
I think there's a better way to do this than linking to the generated files. Can you try `\ref gz::sim::Entity "is just a numeric ID"`?

Reply:
Cool, the links look the same for me.
I did a grep and actually there are no other places using this kind of `\ref` syntax. Is that preferred?

Another pattern that might need fixing is, I did a grep on `classignition` and we actually have quite a few hard URL links that are going to the ignition::gazebo namespace and have "Ignition Gazebo" in the title.

I'm talking about like this page
https://gazebosim.org/api/sim/8/resources.html
the PluginInfo links to this hard URL https://gazebosim.org/api/gazebo/7/classignition_1_1gazebo_1_1ServerConfig_1_1PluginInfo.html
and should instead go to
https://gazebosim.org/api/sim/8/classgz_1_1sim_1_1ServerConfig_1_1PluginInfo.html

They should be changed to at least relative links, if not `\ref`. But not all of them. Some of them, like `classignition_1_1msgs` need to stay, as the gz version doesn't seem to exist?
Like `gz::msgs::EntityFactory` in the [entity creation tutorial](https://gazebosim.org/api/sim/8/entity_creation.html) links to
https://gazebosim.org/api/msgs/6.0/classignition_1_1msgs_1_1EntityFactory__V.html linked from on 
But the gz version of the URL doesn't exist:
https://gazebosim.org/api/msgs/6.0/classgz_1_1msgs_1_1EntityFactory__V.html

Reply:
fdfe8b41d changes the links in this tutorial to `\ref`.

One downside is that `\ref` doesn't play well with text formatting. The bullet heading "Event manager" is lost and I tried a few ways but can't get it back. It may be a known behavior https://github.com/doxygen/doxygen/issues/9023

Reply:
5a0c4a05b makes the hard URLs mentioned above into local links.
I didn't change those to `\ref`, because they are mostly in the migration tutorials, mostly point to the header files as opposed to the class definition. The `\ref` way goes to the class definition.

Reply:
142c447dc is along the same vein - updates more URLs from `ignition` to `gz`, and change some `\ref` links

It also updates to the latest library release version.
I believe I've tested all the new URLs.

Reply:
The usage looks good to me but need to test it once!

Reply:
Okay, I see why I thought other libraries don't work.

gz-msgs, as mentioned above, seems to not follow the pattern of other libraries. A line like [this](https://github.com/gazebosim/gz-sim/pull/2107#discussion_r1324978222) already exists for gz-msgs, but `\ref gz::msgs::EntityFactory`,`\ref gz::msgs::Model`, and `\ref gz::msgs::Light` still don't work. I'm not sure why. I compared the CMakeLists.txt of gz-msgs with others, and it doesn't seem to be missing anything obvious.

Could it be because gz-msgs headers are generated from protobuf?

Reply:
> Could it be because gz-msgs headers are generated from protobuf?

Yes. We used to generate doxygen for the messages (see https://gazebosim.org/api/msgs/8/namespaceignition_1_1msgs.html), but I think we removed them from gz-msgs10. @mjcarroll, do we plan to add them back?